src/runtime-rs/crates/runtimes/virt_container/src/container_manager/process.rs

```diff
@@ -177,12 +186,32 @@ impl Process {
 
             info!(logger, "end wait process exit code {}", resp.status);
 
+            let containers = containers.read().await;
+            let container_id = &process.container_id.container_id;
+            let c = match containers.get(container_id) {
+                Some(c) => c,
+                None => {
+                    error!(
+                        logger,
+                        "Failed to stop process, since container {} not found", container_id
+                    );
+                    return;
+                }
+            };
+
+            if let Err(err) = c.stop_process(&process).await {
```

Comment:
If we had call stop_process here, then there's no need to call it in  wait_process ?

Reply:
Yes. Moving here makes sure that the `stop_process()` is called once the container exited. Therefore, calling it in `wait_process()` is unneccessary.